packages/test/test-end-to-end-tests/src/test/codePropsal.spec.ts

```diff
@@ -37,11 +36,6 @@ function isCodeProposalTestPackage(pkg: unknown): pkg is ICodeProposalTestPackag
 }
 
 describe("CodeProposal.EndToEnd", () => {
-    let driver: ITestDriver;
```

Comment:
Why did you opt to go for scoped declarations of `driver` rather than this?

Reply:
...but there are a few places like in the next file where you still do it in `before`...

Reply:
just reducing member state where easy/possible, as i think it makes the code simpler to understand